### PR TITLE
Add Session model (CU-86791yyvg).

### PR DESCRIPTION
**Pull request by micahdbak**
This PR moves the Session model from BraveButtons and BraveSensor into brave-alert-lib, by combining and aggregating parts from both. It should work as a drop-in replacement for the versions currently implemented in either repositories.

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-86791yyvg Combine Buttons and Sensors databases](https://app.clickup.com/t/86791yyvg)

File: lib/models/Session.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    this.numberOfAlerts = numberOfAlerts
+    this.createdAt = createdAt
+    this.updatedAt = updatedAt
+    this.incidentCategory = incidentCategory
+    this.respondedAt = respondedAt
+    this.respondedByPhoneNumber = respondedByPhoneNumber
+
+    // this is temporary: change these two lines to `this.device = device` when renaming buttons and locations table to devices table in both buttons and sensor
+    this.button = device
+    this.location = device
```
Comment (trisapeace) [inline lib/models/Session.js:18]: Looks like a good temporary measure. during this transition.

File: lib/models/Session.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  }
+
+  // this is specific to buttons: notice that this.alertType is updated
+  incrementButtonPresses(numberOfAlerts) {
+    this.numberOfAlerts += numberOfAlerts
+    this.alertType = ALERT_TYPE.BUTTONS_URGENT
+  }
```
Comment (trisapeace) [inline lib/models/Session.js:25]: Were there tests for this that can also be moved over?

**Review by trisapeace — COMMENTED** [review]
Nice one!

File: lib/models/Session.js (hunk 2 above)
Comment (micahdbak) [reply lib/models/Session.js:25]: There were, that you found in Buttons. I'll move these over :)

**Pull request merged**